argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To use Argo Rollouts and manage progressive delivery, after you install the {gitops-titel} Operator on the cluster, you can create and configure a `RolloutManager` custom resource (CR) instance in the namespace of your choice. You can scope the `RolloutManager` CR for single or multiple namespaces. 
+To use Argo Rollouts and manage progressive delivery, after you install the {gitops-title} Operator on the cluster, you can create and configure a `RolloutManager` custom resource (CR) instance in the namespace of your choice. You can scope the `RolloutManager` CR for single or multiple namespaces. 
```

Comment:
```suggestion
After you install the {gitops-title} Operator on the cluster, you can use Argo Rollouts to manage progressive delivery by creating and configuring a `RolloutManager` custom resource (CR) in a namespace.
```

Reply:
@Dhruv-Soni11 since the suggested changes by @sr1kar99 above are not in scope of this PR, feel free to take your call regarding implementing the suggestion. This is not a mandatory change.

Reply:
I would not like to fix this comment.